.editorconfig

```diff
@@ -0,0 +1,25 @@
+root = true
+
+[*]
```

Comment:
`insert_final_newline = true` and `trim_trailing_whitespace = true` would be nice as well.